lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -110,6 +117,8 @@ const mapStateToProps = (state) => {
   }
   return {
```

Comment:
We should style the transit vehicle overlay to have the arrows attached to the border. I think we should also restore the color hover effect.

Reply:
Good point, fixed in https://github.com/opentripplanner/otp-react-redux/pull/711/commits/89b0e2e3cf8e55d418d0cb0870f36101cc938cdd.